ci/common/test-pygenomeworks.sh

```diff
@@ -51,9 +51,14 @@ pip uninstall -y pygenomeworks
 logger "Create pygenomeworks Wheel package..."
 CUDA_VERSION_FOR_PACKAGE_NAME=$(echo "$CUDA_VERSION" | cut -d"." -f1-2 | sed -e "s/\./_/g")
 if [ "${COMMIT_HASH}" == "master" ]; then
+  # Official Release package
   PYGW_VERSION=$(cat ../VERSION)
 else
-  PYGW_VERSION=$(cat ../VERSION | tr -d "\n")\.dev$(date +%y%m%d) # for nightly build
+  # Development package
+  cd "$WORKSPACE"
+  bash ci/release/update-version.sh #  Promote "patch" version
+  cd "$PYGENOMEWORKS_DIR"
+  PYGW_VERSION=$(cat ../VERSION | tr -d "\n")\.dev$(date +%d) # for nightly build
```

Comment:
what do you think would be a good `VERSION` to use for the _upcoming_ version in the dev branch, especially if we don't know the release date?

Reply:
Official release -> [year][month][0 or **_current promoted_** MICRO version].
nightly release ->[year][month][0 or **_next_** MICRO version].dev[day of the month]
This is the reason `update-version.sh` is called for nightly builds.
This way our nightly packages that are released after some official release in the same month are always one MICRO version ahead. 
I assume this is okay to release a development package without an official release in some months

We should follow these [guidelines](https://packaging.python.org/guides/distributing-packages-using-setuptools/#choosing-a-versioning-scheme).

Reply:
documenting our offline discussion - 
```
master branch -> v2021.02.0
dev branch = next expected release data -> v2021.06.0
nightly builds = v2021.06.0.dev<todays_date>
```